### ISSUE TITLE
Issue loading organization, users in Admin UI

### ISSUE THREAD

**Issue by greycel** [Shuffle/Shuffle#1114]
Hi Everyone!

I've installed Shuffle as per ```https://github.com/Shuffle/Shuffle/blob/main/.github/install-guide.md```, and all containers are up and running. 

Having a couple of issues while accessing the admin page within the shuffle portal
1) The ```organization``` tab gets the error ``` ORGANIZATION ID NOT DEFINED. PLEASE CONTACT US ON HTTPS://SHUFFLER.IO IF THIS PERSISTS LOGOUT. ```, even though ```ORG_ID=Shuffle``` is defined in the ```.env``` file, behavior remains the same even after attempting to re-login.
![2023-05-30_14-08-49](https://github.com/Shuffle/Shuffle/assets/11851504/5aa0aed9-2166-4ebf-840f-b6310c990b6b)

2) The ```users``` tab is not showing or loading available/created users and remains empty
![2023-05-30_14-39-26](https://github.com/Shuffle/Shuffle/assets/11851504/82e2e4d9-52a0-4234-8fc1-1dd41179c290)

     Whereas the frontend & backend container logs show the user creation request was successful.
     ```
    shuffle-frontend  | 172.17.0.1 - - [30/May/2023:09:08:56 +0000] "GET /api/v1/getusers HTTP/1.1" 200 2 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
    shuffle-frontend  | 172.17.0.1 - - [30/May/2023:09:08:56 +0000] "GET /api/v1/users/notifications HTTP/1.1" 200 76 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
   shuffle-frontend  | 172.17.0.1 - - [30/May/2023:09:10:58 +0000] "POST /api/v1/users/register HTTP/1.1" 200 72 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
   shuffle-frontend  | 172.17.0.1 - - [30/May/2023:09:10:59 +0000] "GET /api/v1/getusers HTTP/1.1" 200 2 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"

    shuffle-backend  | 2023/05/30 09:10:58 [INFO] Found 0 (0) user(s) for username user01 in db-connector
    shuffle-backend  | 2023/05/30 09:10:58 [INFO] Updating a user (user01) that has the role user with 0 apps and 1 orgs. Org updater: true
    shuffle-backend  | 2023/05/30 09:10:58 [INFO] Successfully updated org with user user01!
    shuffle-backend  | 2023/05/30 09:10:58 [INFO] user01 Successfully registered.
     ```

Couldn't figure out where the issue could be, all the shuffle services are online and there are no errors logged.
Any advice/guidance on how I can further troubleshoot this?

**Comment by benjaminhernandez21** [comment]
https://github.com/Shuffle/Shuffle/issues/884


> Hi Everyone!
> 
> I've installed Shuffle as per `https://github.com/Shuffle/Shuffle/blob/main/.github/install-guide.md`, and all containers are up and running.
> 
> Having a couple of issues while accessing the admin page within the shuffle portal
> 
>     1. The `organization` tab gets the error `ORGANIZATION ID NOT DEFINED. PLEASE CONTACT US ON HTTPS://SHUFFLER.IO IF THIS PERSISTS LOGOUT.`, even though `ORG_ID=Shuffle` is defined in the `.env` file, behavior remains the same even after attempting to re-login.
>        ![2023-05-30_14-08-49](https://user-images.githubusercontent.com/11851504/241908514-5aa0aed9-2166-4ebf-840f-b6310c990b6b.png)
> 
>     2. The `users` tab is not showing or loading available/created users and remains empty
>        ![2023-05-30_14-39-26](https://user-images.githubusercontent.com/11851504/241910015-82e2e4d9-52a0-4234-8fc1-1dd41179c290.png)
>        Whereas the frontend & backend container logs show the user creation request was successful.
>        ```
>        shuffle-frontend  | 172.17.0.1 - - [30/May/2023:09:08:56 +0000] "GET /api/v1/getusers HTTP/1.1" 200 2 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
>        shuffle-frontend  | 172.17.0.1 - - [30/May/2023:09:08:56 +0000] "GET /api/v1/users/notifications HTTP/1.1" 200 76 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
>        shuffle-frontend  | 172.17.0.1 - - [30/May/2023:09:10:58 +0000] "POST /api/v1/users/register HTTP/1.1" 200 72 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
>        shuffle-frontend  | 172.17.0.1 - - [30/May/2023:09:10:59 +0000] "GET /api/v1/getusers HTTP/1.1" 200 2 "-" "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/113.0.0.0 Safari/537.36"
>        
>        shuffle-backend  | 2023/05/30 09:10:58 [INFO] Found 0 (0) user(s) for username user01 in db-connector
>        shuffle-backend  | 2023/05/30 09:10:58 [INFO] Updating a user (user01) that has the role user with 0 apps and 1 orgs. Org updater: true
>        shuffle-backend  | 2023/05/30 09:10:58 [INFO] Successfully updated org with user user01!
>        shuffle-backend  | 2023/05/30 09:10:58 [INFO] user01 Successfully registered.
>        ```
> 
> 
> Couldn't figure out where the issue could be, all the shuffle services are online and there are no errors logged. Any advice/guidance on how I can further troubleshoot this?

**Comment by greycel** [comment]
I've gone through [#884 ](https://github.com/Shuffle/Shuffle/issues/884). Fortunately, I don't have any issues with permissions or with services, here are permissions on ```shuffle-database``` folder on the host with uid/gid ```1000```
<img width="634" alt="image" src="https://github.com/Shuffle/Shuffle/assets/11851504/981c3518-7da2-4544-8cfb-fd1a9b355106">

and permissions for ```data``` folder in the OpenSearch container
<img width="789" alt="image" src="https://github.com/Shuffle/Shuffle/assets/11851504/d93a8563-69e4-4af6-88e6-725ef00399bc">

All the services running without any errors
<img width="1480" alt="image" src="https://github.com/Shuffle/Shuffle/assets/11851504/d36982ec-ab2c-49d5-a261-4cc9ded0c5ed">


Couldn't figure out why the organization & users tabs in the /admin UI aren't able to load the data...

**Comment by frikky** [comment]
Hey, and sorry for the trouble with this!

It seems like there was a problem that came up where a user sometimes isn't assigned an organization, even though the organization is made. This had to do with storage in Opensearch going wrong with bad timing it seems like. We also discovered a semi-related issue where an ID is not made for organizations, making it so that you get a new org in opensearch each time Shuffle is restarted.

We've made a self-fixing mechanism for this, meaning if you log in to the new version we'll be releasing tonight for Shuffle (as 1.2.0/latest), then this is sorted.
